packages/components/organisms/f-registration/src/components/Registration.vue

```diff
@@ -32,7 +32,6 @@
                 @submit.prevent="onFormSubmit">
                 <section
                     id="error-summary-container"
-                    class="is-visuallyHidden"
```

Comment:
Quick (probably dumb) question – isn't this summary box only for screen readers? And if so, what hides it if this class is removed?